### ISSUE TITLE
DMG audio comparison

### ISSUE THREAD

**Issue by ahihi** [MouseBiteLabs/Game-Boy-DMG-Color#5]
hi, love this project! like many gameboy musicians, i use "prosound" modded DMGs for their clean and bassy audio, so im very curious how the DMGC compares. would it be possible for you to add a recording and spectrum from one of these?

**Comment by MouseBiteLabs** [comment]
I have one here!

https://github.com/MouseBiteLabs/Game-Boy-DMG-Color/tree/main#dmgc-audio-pro-sound

**Comment by ahihi** [comment]
is that not from the DMGC, rather than DMG?

**Comment by MouseBiteLabs** [comment]
Oh, you want a DMG pro-sound to compare. I actually don't have any
pro-sound modded DMGs right now but I can put it on my to-do list.

**Comment by ahihi** [comment]
yes! sorry, i could have been clearer about that :) looking forward to it.
